validate_md_links.py

```diff
@@ -95,6 +95,11 @@ def main():
                 line = md_files[path][:match.start()].count('\n') + 1
                 sys.stderr.write(f'{path}:{line}: error: {message}\n')
                 ok = False
+        for (i, line) in enumerate(md_files[path].split('\n')):
+            if ' -- ' in f' {line} ':
+                sys.stderr.write(
+                    f'{path}:{i+1}: error: replace -- with &mdash;\n')
+                ok = False
     sys.exit(0 if ok else 1)
```

Comment:
`sys.exit(1 * ok)`! _Embrace_ the weakly-typed heresy!

<sub><sub>do not actually do this</sub></sub>

Reply:
that would be `1-ok`, right?

One alternative representation that actually would make a bit of sense, is to go all-in on integers and rename `ok` to `exitcode`.